src/libertem/io/dataset/base/partition.py

```diff
@@ -73,11 +73,11 @@ def make_slices(cls, shape, num_partitions, sync_offset=0):
         num_frames = shape.nav.size
         if num_partitions > num_frames:
             warnings.warn(
-                "dataset contains fewer frames than partitions, "
-                "setting num_part to 1 to allow processing (use fewer workers?)",
+                f"dataset contains fewer frames than partitions, "
+                f"setting num_partitions to {num_frames} to allow processing",
```

Comment:
Improving my warning message from the last PR on this:

```suggestion
                "dataset contains fewer frames than specified partitions, "
                f"setting num_partitions == num_frames == {num_frames} "
                "to avoid creating empty partitions",
```